synth/base/validator.py

```diff
@@ -138,8 +138,6 @@ def run(self):
         # This loop maintains the validator's operations until intentionally stopped.
         try:
             while True:
-                bt.logging.info(f"step({self.step}) block({self.block})")
```

Comment:
for some reason, this causes HTTP 429 too many request errors. It should not because supposedly the get_current_block has caching. This log is not crucial, let's remove it